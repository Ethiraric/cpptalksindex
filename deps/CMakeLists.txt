add_subdirectory(expected)
add_subdirectory(json)
