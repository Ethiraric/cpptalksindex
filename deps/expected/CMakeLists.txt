add_library(expected INTERFACE)
target_include_directories(expected SYSTEM INTERFACE include)
